util/feedback.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+from starlette.exceptions import HTTPException
+from config.db import get_database
+import logging
+DB_NAME = "underline"
+
+# Given an event id and feedback id, attempt to delete the feedback from the event's comment_ids array
+async def delete_feedback(event_id, feedback_id):
+    db = get_database()
```

Comment:
getting the `db` object here is equally valid to passing it in from `routes` but it's inconsistent (anti-pattern vs. the rest of our codebase)

if we had done it from the start it would be fine, and again is totally valid and equally as good, but it's not following the set call pattern.

consider changing signature to accept the incoming `db` object from `routes` (it's already instantiated there anyway)